setup.py

```diff
@@ -12,15 +12,61 @@
 with open(version_file_path, "r") as version_file:
     version = version_file.read()
 
+
+readme_path = os.path.join(
+    os.path.dirname(__file__),
+    "README.md"
+)
+with open(readme_path, "r") as readme_file:
+    readme = readme_file.read()
+
+
+class CheckVersionCommand(Command):
+    description = (
+        "Check that this package's version matches a user-supplied version"
+    )
+    user_options = [
+        ('expected-version=', "e", 'Compare package version against this value')
+    ]
+
+    def initialize_options(self):
+        self.package_version = version
+        self.expected_version = None
+
+    def finalize_options(self):
+        assert self.expected_version is not None
+        if self.expected_version[0] == "v":
+            self.expected_version = self.expected_version[1:]
+
+    def run(self):
+        if self.expected_version != self.package_version:
+            raise ValueError(
+                f"expected version {self.expected_version}, but this package "
+                f"has version {self.package_version}"
+            )
+
+
 setup(
     name='ipfx',
     version=version,
     description="""intrinsic physiology feature extractor""",
```

Comment:
better to change to:
"Intrinsic Physiology Feature Extractor (IPFX) - tool for computing neuronal features from the intracellular electrophysiological recordings"

Reply:
updated it, thanks.